docs/README.rst

```diff
@@ -12,13 +13,29 @@ Pull requests.
 
 Requirements
 ------------
+
+The documentation must be built using Python 3.
+
 The following tools are needed to build the documentation:
 
  - sphinx
+ - sphinx_rtd_theme
 
 On Debian-based systems, you should be able to run::
 
-    sudo apt-get install python-sphinx
+    sudo apt-get install python3-sphinx python3-sphinx-rtd-theme
+
+In a conda environment, you can use::
+
+    conda install sphinx sphinx_rtd_theme
+
+In a Python 3 ``venv``, you should be able to run::
+
+    pip install -U sphinx sphinx_rtd_theme
```

Comment:
I think there is a `docs/requirements.txt`, maybe `pip install --update -r docs/requirements.txt` is even more general ? But I think that's already an improvement as is so I already merged ! Thanks !

Reply:
Good point! I should have thought of that in light of 326f41b and a couple of other things that happened during the sprint yesterday. I'll work on another PR for this.

Reply:
👍